gui/mozregui/skip_chooser.py

```diff
@@ -117,7 +117,7 @@ def build_index(self, item):
         return self.scene.build_range.future_build_infos.index(item.future_build_info)
 
     def choose_next_build(self):
-        if self.exec() == self.Accepted:
+        if self.exec() == SkipDialog.Accepted:
```

Comment:
similar here, why is `self` -> `SkipDialog` ?

Reply:
In this case, this maps to: `DialogCode.Accepted` (same story as `MoveOperation` or `GraphicsItemFlag`). These are just enums.